zanata-war/src/main/java/org/zanata/service/impl/LocaleServiceImpl.java

```diff
@@ -90,12 +91,17 @@ public LocaleServiceImpl(LocaleDAO localeDAO, ProjectDAO projectDAO,
 
     public static Map<LocaleId, String> getLocaleAliasesByIteration(HProjectIteration iteration) {
         Map<LocaleId, String> localeAliases;
```

Comment:
What i meant was you might no need this line at all. 
Just do 
`return iteration.getLocaleAliases();` in line 96 and 
`return iteration.getProject().getLocaleAliases();` in line 98. 

Also, your checkstyle might fail in line 101.